src/v2/Apps/Components/AppShell.tsx

```diff
@@ -23,27 +23,27 @@ const logger = createLogger("Apps/Components/AppShell")
 
 interface AppShellProps {
   children: React.ReactNode
-  match: Match
+  match?: Match
 }
 
 export const AppShell: React.FC<AppShellProps> = props => {
   useAuthIntent()
   useAuthValidation()
 
   const { children, match } = props
-  const routeConfig = findCurrentRoute(match)!
+  const routeConfig = match ? findCurrentRoute(match) : null
   const { isEigen } = useSystemContext()
-  const showNav = !routeConfig.hideNav
-  const showFooter = !isEigen && !routeConfig.hideFooter
-  const appContainerMaxWidth = routeConfig.displayFullPage ? "100%" : null
+  const showNav = !routeConfig?.hideNav
+  const showFooter = !isEigen && !routeConfig?.hideFooter
+  const appContainerMaxWidth = routeConfig?.displayFullPage ? "100%" : null
 
   // Check to see if a route has a onServerSideRender key; if so call it. Used
   // typically to preload bundle-split components (import()) while the route is
   // fetching data in the background.
   useEffect(() => {
-    if (isFunction(routeConfig.onClientSideRender)) {
+    if (isFunction(routeConfig?.onClientSideRender) && match) {
```

Comment:
Curious why all of this stuff got a little more safe? (its fine, but wondering why we needed changes here)

Reply:
This was so that we can use it sans-routing. In this PR I wound up not using the `AppShell` but in thinking about rendering a 'minimal' SSR only shell — we can actually completely re-use it with a few small changes (will follow up)

Reply:
sweet

Reply:
There's some issues with breakpoints in the footer and with login/out status apparently. Going with the minimal header style we use in a few other places, for now.

![](https://static.damonzucconi.com/_capture/3lrNIQDy.png)